frontend/src/types.ts

```diff
@@ -450,15 +450,21 @@ export interface FunnelStepRangeEntityFilter extends EntityFilter {
 
 export type EntityFilterTypes = EntityFilter | ActionFilter | FunnelStepRangeEntityFilter | null
 
-export interface PersonType {
-    id?: number
+export interface ActorType {
```

Comment:
This is the wrong way to approach this.

Rather do it as following:

```
interface TypeA {
  type: 'a'

}

interface TypeB {
  type: 'b'

}

type Union = TypeA | TypeB
```

You will need to repeat some properties between the two but it will set up things in a way where typescript can discriminate the two if you have an if on `.type`.